src/components/header-block/header-block.stories.tsx

```diff
@@ -1,18 +1,9 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
```

Comment:
`ComponentMeta` is not used in this file anymore. Please remove it :pray:

I recommend to always look at your own PR to make sure you didn't forget something obvious. You would've probably seen this notification:
![image](https://user-images.githubusercontent.com/7970458/143901064-dc5a61fc-cd63-4176-bb4e-b1a073d4d7cb.png)

```suggestion
import { ComponentStory } from '@storybook/react';
```

Reply:
Makes sense, thank you